x/genutil/client/cli/init.go

```diff
@@ -163,10 +163,6 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 			genDoc.Validators = nil
 			genDoc.AppState = appState
 
-			if serverCtx.DefaultConsensusParams != nil {
-				genDoc.ConsensusParams = serverCtx.DefaultConsensusParams
-			}
```

Comment:
it looks like we accidently removed our ability to set default consensus parameters